domains/auth/Auth.API/API/Services/OidcProviders/SignaturGruppen.cs

```diff
@@ -1,21 +1,25 @@
+using System.Text.Json;
 using API.Configuration;
 using API.Models;
 using Microsoft.Extensions.Options;
 using System.Text.Json;
 
-namespace API.Services;
+namespace API.Services.OidcProviders;
 
 public class SignaturGruppen : IOidcProviders
 {
     readonly IOidcService oidcService;
     private readonly AuthOptions authOptions;
     readonly ILogger<SignaturGruppen> logger;
+    private readonly HttpClient httpClient;
 
-    public SignaturGruppen(ILogger<SignaturGruppen> logger, IOidcService oidcService, IOptions<AuthOptions> authOptions)
+    public SignaturGruppen(ILogger<SignaturGruppen> logger, IOidcService oidcService, IOptions<AuthOptions> authOptions,
+        HttpClient httpClient)
```

Comment:
Fixed in 8610fe0

Reply:
This will be fixed when we run an editorconfig in [762](https://app.zenhub.com/workspaces/fenris---team-board-616bc40121d71900140955f8/issues/energinet-datahub/energy-origin-issues/762)